tests/unit/factories.py

```diff
@@ -210,6 +213,18 @@ class Meta:  # pylint: disable=too-few-public-methods
     service_config: builder.ExternalServiceConfig = ExternalServiceConfigFactory()
 
 
+class ScriptConfigFactory(factory.Factory):
+    """Image builder script configuration factory."""  # noqa: DCO060
```

Comment:
Can you have an explanatory comment for the ignore?

Reply:
The ignore is repeated for each factory classes and a general comment about the ignores has been documented at the top of the file.